examples/spark-pi-py.yaml

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
Missing the header `Copyright 2018 Google LLC`, which is required for repos hosted under GoogleCloudPlatform.